appsembler/edx-theme
# Move Gymnasium production to new CMS service

attn: @tkeemon 

# What this PR does
- Moves gymnasium's CMS source from the old github pages URLs to gymcms.xyz - with some fancy new mako syntax 

# Dependencies
- This change requires an `APPSEMBLER_FEATURES.ENVIRONMENT` variable to be set via `lms.env.json` or ansible, just like on staging - the value for production should be `production`.

# To test that this was a successful update
after compiling the theme and running the platform:
- the site shouldn't look any different visibly, but if you view the page source for compiled pages in the browser, you'll see comments like this:
![image](https://user-images.githubusercontent.com/1844496/37807121-2818aca4-2e1a-11e8-9313-71f461eafcad.png)
Which indicate content is being pulled from our new CMS.  If they're showing up, everything is :100:.  If not, please roll this PR back for the time being, and we can investigate what's gone wrong.
- See files changed in this PR for which templates use these CMS imports - there are a few scattered over most pages of the site, in various capacities.



# Timeline 
- Once the ansible changes are in place, this should be good to go
